diff --git a/node_modules/react-native-pin-view/index.js b/node_modules/react-native-pin-view/index.js
index e7b3ce0..fa1f31a 100644
--- a/node_modules/react-native-pin-view/index.js
+++ b/node_modules/react-native-pin-view/index.js
@@ -139,6 +139,9 @@ const PinView = React.forwardRef(
           setInput("")
         }
       },
+      initValue: (pin) => {
+          setInput(pin)
+      },
     }
 
     const onButtonPressHandle = (key, value) => {
